packages/Icon/stories/examples/Test.js

```diff
@@ -44,6 +45,7 @@ const ExampleStory = () => (
         }}
       />
       , photo booth small batch poke. <ArrowIcon />
+      <EllipsisIcon />
```

Comment:
Can you please boy scout this story.
Please dont name it Test.js
Maybe make a table with each icon on a line, the name in the first cell, the icon in the second.

Reply:
I agree this story file does have an odd approach to demonstrate the icons, I'm not aware why it was decided to build it out like this though. Perhaps @mikrotron knows more. I would like to come back to refactor this file/ icons after some discussion so perhaps I'll address in a follow up PR

Reply:
Yeah, I made this story as a quick and dirty way to verify the component was working. But deliberately deferred on making a "proper" story until I figure out a consistent approach for our Paprika Storybook (which I did during the HackFest).  I think we need a story that automatically displays all of the icons, rather than something we have to manually update, but that's beyond the scope of this PR.

Reply:
Ok, lets add this component now, and we can address the icons story in follow up issue.